### PR TITLE
docs: open questions for policy-exception

### PR DESCRIPTION
**Pull request by Kavinjsir**

### CODE REVIEW

File: proposal/policyexception-interfaces.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Interfaces Design for operating PolicyExceptions in Kyverno
+
+```go
+type PolicyExceptionInterface interface {
+	metav1.Object
```
Comment (Kavinjsir) [inline proposal/policyexception-interfaces.md:5]: What is this `metav1.Object` for?

File: proposal/policyexception-interfaces.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+	// get finds all excluded resources under a specific policy-rule
+	get(string) []kyvernov2beta1.MatchResources
+}
+
+
+type PolicyExceptionCache struct {
+	store PolicyExceptionStore
+  // policyName1-ruleName1 : [exclude1, exclude2,...]
+	// policyName2-ruleName2: [exclude1, exclude2...]
+	exceptions map[string][]kyvernov2beta1.MatchResources
```
Comment (Kavinjsir) [inline proposal/policyexception-interfaces.md:36]: I think it is better to have a place to store the raw data of `PolicyExceptions`.

Then, you may design methods to get `MatchResources` if it is necessary.

File: proposal/policyexception-interfaces.md (hunk 1 above)
Comment (Eileen-Yu) [reply proposal/policyexception-interfaces.md:5]: For calling k8s API?
would import 
```
metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment (Kavinjsir) [reply proposal/policyexception-interfaces.md:5]: Say I'm the informer, can I use this interface to implement `AddFunc`? (And how if necessary...)

**Pull request closed without merging**